Core/src/org/sleuthkit/autopsy/contentviewers/SQLiteViewer.java

```diff
@@ -64,6 +69,11 @@
     SQLiteTableView selectedTableView = new SQLiteTableView();
 
     private SwingWorker<? extends Object, ? extends Object> worker;
+    
+    
+    //private final SleuthkitCase sleuthkitCase = Case.getCurrentCase().getSleuthkitCase();
+    //private final Services services = new Services(sleuthkitCase);
+    //private final FileManager fileManager = services.getFileManager();
```

Comment:
Please remove these commented out fields.

Reply:
Removed the commented code.